src/wavesurfer.js

```diff
@@ -1168,6 +1173,16 @@ export default class WaveSurfer extends util.Observer {
         this.drawBuffer();
     }
 
+    /**
+     * When splitChannels is true, hides channels from being drawn on the waveform.
+     *
```

Comment:
i'm not able to visualize what happens here when reading that sentence. Can you rephrase/elaborate bit more? maybe also add an `@example` with small example as well to demonstrate it's use.

Reply:
Does this make better sense now?

Reply:
much better!